src/main/java/mate/project/controller/BookController.java

```diff
@@ -1,5 +1,6 @@
 package mate.project.controller;
 
+import jakarta.validation.Valid;
```

Comment:
The @Valid annotation is used to perform bean validation on the request body. Ensure that corresponding validation annotations are present in the CreateBookRequestDto class.